### PR TITLE
Add wdspec test for BiDi network.addIntercept with invalid arguments

### PR DESCRIPTION
**Pull request by juliandescottes**

### CODE REVIEW

**Comment by juliandescottes** [comment]
cc @thiagowfx here's what I have locally, although it really on covers invalid arguments for addIntercept. I can't really test actual interception yet.

File: webdriver/tests/bidi/network/add_intercept/invalid.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+@pytest.mark.parametrize("value", ["abc/com/", "abc?com", "abc#com", "abc:com"])
+async def test_params_url_patterns_pattern_hostname_invalid_value(bidi_session, value):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.network.add_intercept(
+            phases=["beforeRequestSent"],
+            url_patterns=[{"type": "pattern", "hostname": value}],
+        )
+
+
+@pytest.mark.parametrize("value", ["abcd", "-1", "80 ", "1.3", ":80"])
```
Comment (thiagowfx) [inline webdriver/tests/bidi/network/add_intercept/invalid.py:147]: maybe add 65536 as well? (one value above the maximum)

**Review by thiagowfx — APPROVED** [review]

File: webdriver/tests/bidi/network/add_intercept/invalid.py (hunk 1 above)
Comment (juliandescottes) [reply webdriver/tests/bidi/network/add_intercept/invalid.py]: Good point! Added.

**Review by thiagowfx — APPROVED** [review]

**Comment by juliandescottes** [comment]
Thanks for the review here! We are going to land this in mozilla-central in the end, but it's almost identical to what was uploaded here.

**Comment by thiagowfx** [comment]
Thanks! Could you ping here once it's landed downstream?

**Comment by juliandescottes** [comment]
> Thanks! Could you ping here once it's landed downstream?

Sure, should happen today.

**Comment by juliandescottes** [comment]
@thiagowfx the tests for addIntercept were merged in https://github.com/web-platform-tests/wpt/pull/41492
I also landed similar tests for removeIntercept, but the sync PR still needs to be created for those.

**Comment by thiagowfx** [comment]
Thanks, already pulling them in today to our repo

**Pull request closed without merging**